examples/07-periodic_hills/periodic_hills.prm

```diff
@@ -53,7 +53,8 @@ end
 #---------------------------------------------------
 subsection post-processing
     set calculate average velocities    = true
-    set initial time 		        = 0.5
+    set calculate reynolds stress       = true
+    set initial time 		        = 100
```

Comment:
This would be how it would be done in the real case? For the tutorial it would be better to have it done sooner, even if this will not be the "real" way it is done. The idea  is still to be able to run the tutorial for a limited amount of time like 30-minutes or 1 hour.

Reply:
I should be more careful about committing the final .prm in my last push, my bad!